static/js/hotkey.js

```diff
@@ -605,6 +606,20 @@ exports.process_hotkey = function (e, hotkey) {
         return false;
     }
 
+    if (event_name === "next_unread_topic") {
+        var current_variables = narrow.current_stream_topic();
+        var next_variables = topic_generator.next_topic(current_variables.stream, current_variables.topic);
+        if (!next_variables.topic) {
+          return true;
```

Comment:
Yes, I will take care of that. I just changed my editor, so haven't configured the tab. I thought of doing this just before the final merge.